repair/repair.cc

```diff
@@ -1015,6 +1017,15 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
     auto id = repair_tracker().next_repair_command();
     rlogger.info("repair[{}]: starting user-requested repair for keyspace {}, repair id {}, options {}", id.uuid, keyspace, id.id, options_map);
 
+    task_status.id = id.uuid;
+    task_status.state = task_manager::task_state::running;
+    task_status.is_abortable = true;
```

Comment:
I think that this should be implemented using a virtual method, defaulted to return 'false'.
and the repair task can override it and return true (other specializations may return true in some cases but not all, e.g. if a global task can be aborted but not per-shard)